RentACar/src/app/service/user.service.ts

```diff
@@ -20,6 +22,14 @@ export class UserService {
 
   register(user: User) {
     return this.http.post(userServiceApi + 'user/register', user)
+  }     
+
+  grabUser(): Observable<User> {
+    return this.http.get<User>('http://localhost:8082/user/' + sessionStorage.getItem('username'));
```

Comment:
use `userServiceApi` const instead of localhost path

Reply:
solved